src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -345,4 +350,140 @@ public function theFirstOrderShouldHaveNumber(string $number)
+     */
+    public function iSpecifyFilterDateFromAs($dateTime)
+    {
+        $this->client->addFilter('checkoutCompletedAt[after]', $dateTime);
+    }
+
+    /**
+     * @When I specify filter date to as :dateTime
+     */
+    public function iSpecifyFilterDateToAs($dateTime)
+    {
+        $this->client->addFilter('checkoutCompletedAt[before]', $dateTime);
+    }
+
+    /**
+     * @When I filter
+     */
+    public function iFilter()
+    {
+        $this->client->filter();
+    }
+
+    /**
+     * @Then I should see :count orders in the list
+     */
+    public function itShouldHaveAmountOfItemsInTheList($count)
+    {
+        $numberOfItems = $this->responseChecker->getValue($this->client->getLastResponse(), 'hydra:totalItems');
+        Assert::eq($numberOfItems, $count);
+    }
+
+    /**
+     * @Then I should not see an order with :number number
+     */
+    public function iShouldNotSeeAnOrderWithNumber($number)
+    {
+        Assert::false($this->responseChecker->hasItemWithValue($this->client->getLastResponse(), 'number', $number));
+    }
+
+    /**
+     * @When I choose :channel as a channel filter
+     */
+    public function iChooseChannelAsAChannelFilter(ChannelInterface $channel)
+    {
+        $this->client->addFilter('channel.code', $channel->getCode());
+    }
+
+    /**
+     * @When /^I filter by (product "([^"]+)")$/
+     */
+    public function iFilterByProduct(ProductInterface $product): void
+    {
+        $this->client->addFilter('items.variant.product[]', $product->getId());
+
+        $this->iFilter();
+    }
+
+    /**
+     * @When /^I filter by (products "([^"]+)" and "([^"]+)")$/
+     * @param ProductInterface[] $products
+     */
+    public function iFilterByProducts(array $products): void
+    {
+        foreach ($products as $productName) {
+            $this->client->addFilter('items.variant.product[]', $productName->getId());
+        }
+
+        $this->iFilter();
+    }
+
+    /**
+     * @When I choose :shippingMethod as a shipping method filter
+     */
+    public function iChooseMethodAsAShippingMethodFilter(ShippingMethodInterface $shippingMethod)
+    {
+        $this->client->addFilter('shipments.method.code', $shippingMethod->getCode());
+    }
+
+
+    /**
+     * @When I choose :currency as the filter currency
+     */
+    public function iChooseAsTheFilterCurrency(CurrencyInterface $currency)
+    {
+        $this->client->addFilter('currencyCode', $currency->getCode());
+    }
+
+    /**
+     * @Then I should not see any orders with currency :currency
+     */
+    public function iShouldNotSeeAnyOrdersWithCurrency(CurrencyInterface $currency)
+    {
+        Assert::false($this->responseChecker->hasItemWithValue($this->client->getLastResponse(), 'currencyCode', $currency->getCode()));
+    }
+
+    /**
+     * @When I specify filter total being greater than :total
+     */
+    public function iSpecifyFilterTotalBeingGreaterThan($total)
```

Comment:
float/string $total?

```suggestion
    public function iSpecifyFilterTotalBeingGreaterThan($total): void
```